cmake/modules/version.rc.in

```diff
@@ -0,0 +1,34 @@
+#include <winresrc.h>
+
+VS_VERSION_INFO VERSIONINFO
+    FILEVERSION @MIRALL_VERSION_MAJOR@,@MIRALL_VERSION_MINOR@,@MIRALL_VERSION_PATCH@,@OC_RC_VERSION_BUILD@
+    PRODUCTVERSION @MIRALL_VERSION_MAJOR@,@MIRALL_VERSION_MINOR@,@MIRALL_VERSION_PATCH@,@OC_RC_VERSION_BUILD@
+    FILEFLAGSMASK VS_FFI_FILEFLAGSMASK
+#ifdef _DEBUG
+    FILEFLAGS VS_FF_DEBUG
+#else
+    FILEFLAGS 0x0L
+#endif
+    FILEOS VOS__WINDOWS32
+    FILETYPE @OC_RC_TYPE@
+    FILESUBTYPE VFT2_UNKNOWN
+    BEGIN
+        BLOCK "StringFileInfo"
+        BEGIN
+            BLOCK "040904E4"
+            BEGIN
+                VALUE "CompanyName", "@APPLICATION_VENDOR@\0"
+                VALUE "LegalCopyright", "Copyright (C) 2014-@MIRALL_VERSION_YEAR@ @APPLICATION_VENDOR@\0"
```

Comment:
The start date is from the shell integration .rc files.
Maybe we want to use 2011 or don't provide a start date.

Reply:
@hodyroff